### DISCUSSION TITLE
Scale

### DISCUSSION THREAD

**Discussion by GerhardtB** [aquacropos/aquacrop#75] [Q&A]
Hi, at what scale does the point simulation run? 

Thanks,

Regards
Gerhardt

**Comment by thomasdkelly** [comment]
Hi @GerhardtB 

Is this a question regarding AquaPlan. If so ill move it into the aquaplan category of the discussion forum. 

Currently when selecting a point inside the US the app uses DAYMET api which has a 1kmx1km resolution. When selecting outside the US, it uses NASA-POWER api which has a 0.5degreex0.5degree resolution

Tom
